### PR TITLE
Add nekofetch

### PR DESCRIPTION
**Pull request by brennanleblanc0**

### CODE REVIEW

**Comment by BBaoVanC** [comment]
bruh

**Comment by syns** [comment]
bruh

**Comment by proprdev** [comment]
bruh

**Comment by brennanleblanc0** [comment]
bruh

File: nekofetch.mk [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	$(call EXTRACT_TAR,v$(NEKOFETCH_VERSION).tar.gz,nekofetch-$(NEKOFETCH_VERSION),nekofetch)
+
+ifneq ($(wildcard $(BUILD_WORK)/nekofetch/.build_complete),)
+nekofetch:
+	@echo "Using previously built nekofetch."
+else
+nekofetch: nekofetch-setup
+	+$(MAKE) -C $(BUILD_WORK)/nekofetch install \
+		PREFIX=$(BUILD_STAGE)/nekofetch$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)
+	# TODO: Separate DESTDIR from PREFIX when fix is added upstream
```
Comment (BBaoVanC) [inline nekofetch.mk:20]: Merged proprdev/nekofetch#8 now

**Pull request merged**